core/ledger/kvledger/txmgmt/statedb/statecouchdb/cache_value.proto

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: Apache-2.0
 
 syntax = "proto3";
 
-option go_package = "github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb";
+option go_package = "github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statecouchdb";
 
 package statedb;
```

Comment:
I don't think it truly matters, but should this be `statecouchdb` as well?

Reply:
Thanks for catching this. It was never intended. I missed out this change.

Reply:
Done.